File: Modul-1/Soal-10/PRAK110-2410817220012-NOOR-KHALISA.py
import math

# Alas dan tinggi segitiga
alas = 5  # dalam cm
tinggi = 12  # dalam cm

# Sisi miring (sisi C) menggunakan Pythagoras
sisi_a = tinggi
sisi_b = int(math.sqrt((alas ** 2) + (tinggi ** 2)))  # Casting ke int
sisi_c = alas

# Keliling segitiga
keliling = sisi_a + sisi_b + sisi_c

# Luas segitiga
luas = (alas * tinggi) // 2  # Pembagian bulat

# Output
print("Diketahui:")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm\n")
print("Jawab:")
print(f"Sisi A = {sisi_a} cm")
print(f"Sisi B = {sisi_b} cm")
print(f"Sisi C = {sisi_c} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm")

File: Modul-1/Soal-10/PRAK110-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
#include <math.h>

int main() {
    // Alas dan tinggi segitiga
    int alas = 5;  // dalam cm
    int tinggi = 12;  // dalam cm

    // Sisi miring (sisi C) 
    int sisiA = tinggi;
    int sisiB = (int) sqrt((alas * alas) + (tinggi * tinggi));
    int sisiC = alas;  
    
    // Keliling segitiga
    int keliling = sisiA + sisiB + sisiC;

    // Luas segitiga
    int luas = (alas * tinggi) / 2;

    // Output
    printf("Diketahui:\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab:\n");
    printf("Sisi A = %d cm\n", sisiA);
    printf("Sisi B = %d cm\n", sisiB);
    printf("Sisi C = %d cm\n", sisiC);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);

    return 0;
}